### PR TITLE
Block Entity Renderers article

### PR DESCRIPTION
**Pull request by natri0**
Describes how to add a BlockEntityRenderer, basic MatrixStack usage (make the text being drawn be on the top side of the block), how to measure & draw text.

### CODE REVIEW

File: develop/blocks/block-entity-renderer.md
Comment (IMB11) [inline develop/blocks/block-entity-renderer.md]:
```suggestion
Sometimes, using Minecraft's model format is not enough. If you need to add dynamic rendering to your models you will need to use a `BlockEntityRenderer`.
```

File: develop/blocks/block-entity-renderer.md
Comment (IMB11) [inline develop/blocks/block-entity-renderer.md]:
```suggestion
First, we need to create a `BlockEntityRenderer` for our `CounterBlockEntity`. 

When creating a `BlockEntityRenderer` for the `CounterBlockEntity`, it's important to place the class in the appropriate source set, such as `src/client/`, if your project uses split source sets for client and server. Accessing rendering-related classes directly in the `src/main/` source set is not safe because those classes might be loaded on a server. 
```

File: develop/blocks/block-entity-renderer.md
Comment (IMB11) [inline develop/blocks/block-entity-renderer.md]:
```suggestion
Also, by including a constructor like this, it becomes possible to use the constructor as a `BlockEntityRendererFactory` functional interface itself:
```

File: develop/blocks/block-entity-renderer.md
Comment (IMB11) [inline develop/blocks/block-entity-renderer.md]: Not needed, just tell users to register the factories in their client mod entrypoint.

**Review by IMB11 — CHANGES_REQUESTED** [review]
Very good and concise. Just a few minor nitpicks.

**Review by IMB11 — DISMISSED** [review]

**Comment by IMB11** [comment]
LGTM

**Comment by its-miroma** [comment]
Does this come from the Wiki or is it OC?

**Comment by IMB11** [comment]
Looks like it's OC - the wiki's example is an item pedestal, could reuse the wiki example at a later date for a "Rendering Items/Blocks in the world" page of some sort.

**Comment by IMB11** [comment]
Just waiting for #234 to merge before merging :)

**Review by IMB11 — APPROVED** [review]

**Comment by IMB11** [comment]
LGTM

**Comment by its-miroma** [comment]
awaiting the check

**Review by its-miroma — APPROVED** [review]

**Pull request merged**
